scripts/release/check_sync_criteria.py

```diff
@@ -54,7 +54,7 @@ def main(argv=sys.argv[1:]):
     return 0 if success else 1
 
 
-def check_sync_criteria(
+def get_missing_packages(
```

Comment:
This function is called `get_missing_packages` but has a multiple return value that include `binary_packages`. That seems worth a docblock or a revisited name to me.